File: C/queue/queue.c
#include<stdio.h>
#include<conio.h>
#define MAX 100
int queue[MAX];
int front=0;
int rear=-1;
void enqueue()
{
    int data;
    if(rear==MAX-1)
    {
        printf("Queue overflow \n\n");
        return ;
    }
    printf("Enter element to be inserted : ");
    scanf("%d",&data);
    rear++;
    queue[rear]=data;
    printf("Element inserted to queue. .\n\n");
}
void dequeue()
{
    if(front>rear)
    {
        printf("Stack underflow. \n\n");
        return ;
    }
    printf("Element removed : %d\n\n",queue[front]);
    front++;
}
void display()
{
    int i;
    if(front>rear)
    {
        printf("No elements to display. \n\n");
        return ;
    }
    printf("Queue : ");
    for(i=front;i<=rear;i++)
    {
        printf("%d \t",queue[i]);
    }
    printf("\n\n");
}
int main()
{
int ch;
	do
	{
		printf("1. Insert an element to queue \n 2. Remove an element from queue \n 3. Display elements of queue \n 4 . Exit");
		printf("\nEnter choice : ");
		scanf("%d",&ch);
		switch(ch)
		{
			case 1 : enqueue();
					 break;
			case 2 : dequeue();
					 break;
			case 3 : display();
					 break;
			case 4 : exit(0);
			default : printf("Invalid choice");
					break;
		}
	}while(ch!=4);
    getch();
    return 0;
}

File: python/queue/queue.py
class Queue():
    def __init__(self):
        self.max = 100
        self.queue = [0]*self.max
        self.front = 0
        self.rear = -1

    def enqueue(self,x):
        if self.rear==self.max:
            print('No more elements can be added')
            return
        self.rear += 1
        self.queue[self.rear] = x
        print(f'Element {x} got inserted')

    def dequeue(self):
        if self.rear==-1 or self.front>self.rear:
            print('No element to be remove')
            return
        print(f'Element {self.queue[self.front]} removed')
        self.front += 1
    def display(self):
        for i in range(self.front,self.rear+1):
            print(self.queue[i],end=' ')
        print()

q = Queue()
q.enqueue(1)
q.enqueue(2)
q.enqueue(3)
q.enqueue(4)
q.enqueue(5)
q.display()
q.dequeue()
q.display()